index.js

```diff
@@ -0,0 +1,513 @@
+/**
+ * @Discord-Bot : Cuddles
+ */
+
+const Discord = require('discord.js');
+const { prefix, token, giphyToken, author_id } = require('./config.json'); // config related variables
+const { animeGifs, fGifs } = require('./arraylist.json'); // Links to all giphy gifs are stored here
+const GphApiClient = require('giphy-js-sdk-core');
+giphy = GphApiClient(giphyToken);
```

Comment:
declare this as a `const` as it will never change and its your giphy api client

Reply:
Have done so, thanks